cts.schema.json

```diff
@@ -32,6 +32,9 @@
         "result": {
           "$ref": "#/$defs/test_case_result"
         },
+        "results": {
+          "$ref": "#/$defs/test_case_results"
+        },
```

Comment:
@gregsdennis I would like to specify that "result" and "results" are mutually exclusive, but I don't think JSON Schema supports that. Right?

Reply:
You can.  Where `result` is required, add `"properties": { "results": false } }`, and vice versa.  This makes all valid in `results` invalid.  It's effectively a "don't use this".

Reply:
@gregsdennis Perfect. Thanks.